modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -16991,6 +16991,7 @@ workflow-definitions=Workflow Definitions
 workflow-deleted-successfully=Workflow deleted successfully.
 workflow-index-actions=Workflow Index Actions
 workflow-indexes=Workflow Indexes
+workflow-instance-not-found=Workflow Instance not found
```

Comment:
@ramonmsp To follow BChan standards, the language key needs to be a title (Workflow Instance Not Found) or a sentence (Workflow instance not found.). In this case, I think we should go with the title pattern.